requirements-dev.txt

```diff
@@ -1,11 +1,30 @@
 # Testing Requirements
```

Comment:
Are all of these additions really necessary? Couldn't we just add ruff?

Reply:
Updated and removed other packages